format/view-spec.md

```diff
@@ -97,7 +97,10 @@ Summary is a string to string map of metadata about a view version. Common metad
 
 View definitions can be represented in multiple ways. Representations are documented ways to express a view definition.
 
-A view version can have more than one representation. All representations for a version must express the same underlying definition. Engines are free to choose the representation to use.
+A view version can have more than one representation. All representations for a version must express the same underlying definition. Engines are free to choose the representation to use. For the table identifiers in the view definitions to be interoperable across engines, the following must be true:
```

Comment:
> For the table identifiers in the view definitions to be interoperable across engines, the following must be true:

is below a little more clear?
```
For interoperability across engines, the following rules are recommended for table identifiers.
```

Reply:
I am fine with paraphrasing, as long as the spec is firm about the rules (e.g., must vs recommended).